_config.yml

```diff
@@ -20,16 +20,10 @@ social:
   facebook: "https://www.facebook.com/FarsetLabs"
   twitter: "https://www.twitter.com/FarsetLabs"
   flickr: "https://www.flickr.com/groups/farset_labs"
-  blogfeed: "https://blog.farsetlabs.org.uk/feed/"
   twitch: "https://www.twitch.tv/farsetlabs"
   instagram: "https://www.instagram.com/farsetlabs"
-links:
-  - name: Blog
```

Comment:
I guess if we merge this we're committing in principle to no longer having a blog.

Reply:
@robertwpaul Opinionated, right? 😁 Thoughts?

Reply:
I think we haven't used it for a while and if we resurrect we can always add the links back in.